src/libs/blueprint/conduit_blueprint_mcarray.cpp

```diff
@@ -346,6 +346,18 @@ verify(const std::string &/*protocol*/,
 //----------------------------------------------------------------------------
 bool verify(const conduit::Node &n,
             Node &info)
+{
+    // TODO(JRC): Make the second parameter the largest value for the 'index_t'
+    // type (unfortunately this isn't trivial to get because 'index_t' can be
+    // signed or unsigned depending on Conduit's build options).
+    return mlarray::verify(n, info, 0, 10000);
```

Comment:
This needs to change and be made more general before this pull request is merged. Any advice on how to properly create a version of `std::numeric::limits<T>::max` that supports the build-configurable type `index_t` would be appreciated.

Reply:
@xjrc  you should be able to use that template fun with index_t, i quickly tried:

 
```
   #include <limits>
    std::cout << std::numeric_limits<conduit::index_t>::max() << std::endl;
```

Maybe you just hit a typo? (`numeric_limits` (ok) vs `numeric::limits` (not ok) ?)

Reply:
I'm not really sure why it didn't work before when I tried it; I seem to remember compiling and running the code with `std::numeric_limits` expressions, so I must've messed it in some other novel way. In any case, it seems to be working now; thank you for the advice!